### PR TITLE
Modify AsociacionDrupalES-DrupalCampSpain project to deploy 2022.drupalcamp.es

### PR DESCRIPTION
**Pull request by KrisMagjistari**

### CODE REVIEW

File: projects/AsociacionDrupalES-DrupalCampSpain/ansible/extra-vars.yml [hunk 1]
```diff
@@ -15,10 +15,9 @@ drush_sanitize_mail: 'drupalcamp+user_%uid@drupal.es'
...
-  web/sites/default/private: "{{ shared_dir }}"
-
-shared_files:
-  web/sites/default/settings.local.php: "{{ shared_file }}"
+#shared_dirs:
+#  web/sites/default/files: "{{ shared_dir }}"
+#  web/sites/default/private: "{{ shared_dir }}"
 
+#shared_files:
+#  web/sites/default/settings.local.php: "{{ shared_file }}"
```
Comment (KrisMagjistari) [inline projects/AsociacionDrupalES-DrupalCampSpain/ansible/extra-vars.yml:23]: For now there are no shared files or dirs for the webapp so I've commented these lines

File: projects/AsociacionDrupalES-DrupalCampSpain/config.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ pipelines:
       jobFolder: ''
     vars:
       repo:
-        url: 'git@github.com:AsociacionDrupalES/DrupalCampSpain.git'
+        url: 'git@github.com:AsociacionDrupalES/DrupalCampSpain2022.git'
         credentials_id: '20552984-a646-4dfc-b94a-59f26887fd59'
       deploy:
         credentials_id: '02304973-08cf-4390-8edb-7a5c9d940cb8'
```
Comment (KrisMagjistari) [inline projects/AsociacionDrupalES-DrupalCampSpain/config.yml:26]: I have doubts as to whether the credential_id values must be changes or not.

**Comment by KrisMagjistari** [comment]
Is there anything else to do? The deconfig of de reload-dev-db job I was think of doing it in Jenkins, by disabling the pipeline. Is there a way to do it with keynes?

File: projects/AsociacionDrupalES-DrupalCampSpain/ansible/extra-vars.yml (hunk 1 above)
Comment (jonhattan) [reply projects/AsociacionDrupalES-DrupalCampSpain/ansible/extra-vars.yml:23]: Descoméntalas. Son los shared por defecto de drupal

File: projects/AsociacionDrupalES-DrupalCampSpain/config.yml (hunk 2 above)
Comment (jonhattan) [reply projects/AsociacionDrupalES-DrupalCampSpain/config.yml:26]: No, as far as it works

**Comment by jonhattan** [comment]
he disable of the reload job is temporarily. It is enough to disable it in the jenkins UI

File: projects/AsociacionDrupalES-DrupalCampSpain/ansible/extra-vars.yml (hunk 1 above)
Comment (KrisMagjistari) [reply projects/AsociacionDrupalES-DrupalCampSpain/ansible/extra-vars.yml]: @jonhattan Lo he intentado pero como actualmente es una web estatica de html, el despliegue falla porque los archivos shared no existen en el repositorio. Lo mejor seria dejarlos comentados y despues descomentarlos cuando desplieguen la version con drupal.

Comment (jonhattan) [reply projects/AsociacionDrupalES-DrupalCampSpain/ansible/extra-vars.yml:23]: ok

**Pull request merged**